docs/topics/images.md

```diff
@@ -0,0 +1,372 @@
+```
+
+(image_tag_alt)=
+## More control over the `img` tag
+
+Wagtail provides two shortcuts to give greater control over the `img` element:
+
+**1. Adding attributes to the  {% image %} tag**
+
+Extra attributes can be specified with the syntax `attribute="value"`:
+
+```html+django
+{% image page.photo width-400 class="foo" id="bar" %}
+```
+
+You can set a more relevant `alt` attribute this way, overriding the one automatically generated from the title of the image. The `src`, `width`, and `height` attributes can also be overridden, if necessary.
+
+**2. Generating the image "as foo" to access individual properties**
+
+Wagtail can assign the image data to another variable using Django's `as` syntax:
+
+```html+django
+{% image page.photo width-400 as tmp_photo %}
+
+<img src="{{ tmp_photo.url }}" width="{{ tmp_photo.width }}"
+    height="{{ tmp_photo.height }}" alt="{{ tmp_photo.alt }}" class="my-custom-class" />
+```
+
+```{note}
+The image property used for the `src` attribute is `image.url`, not `image.src`.
+```
+
+This syntax exposes the underlying image Rendition (`tmp_photo`) to the developer. A "Rendition" contains the information specific to the way you've requested to format the image using the resize-rule, i.e. dimensions and source URL. The following properties are available:
+
+### `url`
+
+URL to the resized version of the image. This may be a local URL (such as `/static/images/example.jpg`) or a full URL (such as `https://assets.example.com/images/example.jpg`), depending on how static files are configured.
+
+
+### `width`
+
+Image width after resizing.
+
+### `height`
+
+Image height after resizing.
+
+### `alt`
+
+Alternative text for the image, typically taken from the image title.
+
+### `attrs`
+
+A shorthand for outputting the attributes `src`, `width`, `height` and `alt` in one go:
+
+```html+django
+<img {{ tmp_photo.attrs }} class="my-custom-class" />
+```
+
+### `full_url`
+
+Same as `url`, but always returns a full absolute URL. This requires `BASE_URL` to be set in the project settings.
+
+This is useful for images that will be re-used outside of the current site, such as social share images:
+
+```html+django
+<meta name="twitter:image" content="{{ tmp_photo.full_url }}">
+```
+
+If your site defines a custom image model using `AbstractImage`, any additional fields you add to an image (e.g. a copyright holder) are **not** included in the rendition.
+
+Therefore, if you'd added the field `author` to your AbstractImage in the above example, you'd access it using `{{ page.photo.author }}` rather than `{{ tmp_photo.author }}`.
+
+(Due to the links in the database between renditions and their parent image, you *could* access it as `{{ tmp_photo.image.author }}`, but that has reduced readability.)
+
+## The `attrs` shortcut
```

Comment:
This was probably in the docs already but can you please merge this `attrs` section with the one above on line 187 

Or move the content from line 187 down into this part.

Reply:
Seems like this should just be in the `attrs` section in line 187. Duplicate content.